### PR TITLE
add template for pull requests

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+# Add a reference to related issue
+Fixes: #NUMBER
```
Comment (praiskup) [inline .github/pull_request_template.md:3]: Preferably into the git commit message, so people in command line have the info, too.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+# Add release notes. See https://rpm-software-management.github.io/mock/Release-Notes-New-Entry
+# or add label no-release-notes if you think the change does not deserve a release notes.
```
Comment (praiskup) [inline .github/pull_request_template.md:6]: Contributors that need this kind of info will not have the rights to set this label :-( so I'd probably directly go to "let us know"

File: .github/pull_request_template.md (hunk 1 above)
Comment (xsuchy) [reply .github/pull_request_template.md:3]: updated.

File: .github/pull_request_template.md (hunk 2 above)
Comment (xsuchy) [reply .github/pull_request_template.md]: updated.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+# Add a reference to related issue - preferably in the git commit message
+Fixes: #NUMBER
+
+# Add release notes. See https://rpm-software-management.github.io/mock/Release-Notes-New-Entry
+# or let us know to add `label no-release-notes` if you think the change does
+# not deserve a release notes.
```
Comment (praiskup) [inline .github/pull_request_template.md:7]: nit: s/notes/note/ or /notes/notes entry/ otherwise lgtm

Comment (xsuchy) [reply .github/pull_request_template.md]: Updated.

**Review by praiskup — APPROVED** [review]

**Comment by praiskup** [comment]
Thank you!

**Pull request merged**